activerecord/test/cases/adapters/postgresql/rename_table_test.rb

```diff
@@ -14,6 +13,28 @@ def teardown
   end
 
   test "renaming a table also renames the primary key index" do
+    @connection.create_table :before_rename, force: true
+
+    assert_changes(-> { num_indices_named("before_rename_pkey") }, from: 1, to: 0) do
+      assert_changes(-> { num_indices_named("after_rename_pkey") }, from: 0, to: 1) do
+        @connection.rename_table :before_rename, :after_rename
+      end
+    end
+  end
+
+  test "renaming a table with uuid primary key also renames the primary key index" do
+    @connection.create_table :before_rename, force: true, id: :uuid
+
+    assert_changes(-> { num_indices_named("before_rename_pkey") }, from: 1, to: 0) do
+      assert_changes(-> { num_indices_named("after_rename_pkey") }, from: 0, to: 1) do
+        @connection.rename_table :before_rename, :after_rename
+      end
+    end
+  end
+
+  test "renaming a table with uuid primary key and default also renames the primary key index" do
+    @connection.create_table :before_rename, force: true, id: :uuid, default: -> { "gen_random_uuid()" }
```

Comment:
Since `gen_random_uuid` is already the default function for PostgreSQL >= 9.4, this is exercising the same code path as the previous test.  Perhaps both tests should specify a default, one for `uuid_generate_v4` and one for `gen_random_uuid`?